simplify installation page (#17759)
netdata/netdata: packaging/installer/README.md

# Netdata Agent Installation

Netdata is very flexible and can be used to monitor all kinds of infrastructure. Read more about possible [Deployment guides](/docs/deployment-guides/README.md) to understand what better suites your needs.

## Install through Netdata Cloud

The easiest way to install Netdata on your system is via Netdata Cloud, to do so:

1. Sign up to <https://app.netdata.cloud/>.
2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform.
3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it.

> Upon installation completing successfully, you should be able to see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).

Reviewer: ```suggestion
Once Netdata is installed, you can see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).
```

# Netdata Agent Installation

Netdata is very flexible and can be used to monitor all kinds of infrastructure. Read more about possible [Deployment guides](/docs/deployment-guides/README.md) to understand what better suites your needs.

## Install through Netdata Cloud

The easiest way to install Netdata on your system is via Netdata Cloud, to do so:

1. Sign up to <https://app.netdata.cloud/>.
2. You will be presented with an empty space, and a prompt to "Connect Nodes" with the install command for each platform.
3. Select the platform you want to install Netdata to, copy and paste the script into your node's terminal, and run it.

> Upon installation completing successfully, you should be able to see the node live in your Netdata Space and charts in the [Metrics tab](/docs/dashboards-and-charts/metrics-tab-and-single-node-tabs.md).

Reviewer: Duplicate text

<img width="1298" alt="Screenshot 2024-05-27 at 14 45 23" src="https://github.com/netdata/netdata/assets/22274335/cf5bad35-d678-4065-99e0-c404560eaf89">

Duplicate text

<img width="1298" alt="Screenshot 2024-05-27 at 14 45 23" src="https://github.com/netdata/netdata/assets/22274335/cf5bad35-d678-4065-99e0-c404560eaf89">

Author: let me find the duplicate and see what to do about it

Author: wow, now I get it

Author: I must've pressed the revert button on something and added the previous wording back in

Reviewer: > let me find the duplicate and see what to do about it

I even posted duplicate report of this text!
